### PR TITLE
phpstan fixes (php doc blocks)

### PR DESCRIPTION
**Pull request by williamdes**

### CODE REVIEW

**Comment by williamdes** [comment]
@fredericdelordm some basic fixes

**Review by mralaminahamed — APPROVED** [review]

File: MangoPay/Libraries/ApiOAuth.php [hunk 1]
```diff
@@ -30,6 +30,7 @@ public function CreateToken()
             $rest->AddRequestHttpHeader('x-tenant-id: uk');
         }
         $response = $rest->Request($urlMethod, $requestType, $requestData);
+        /** @var \MangoPay\Libraries\OAuthToken $token */
```
Comment (iulian03) [inline MangoPay/Libraries/ApiOAuth.php:33]: No need for doc in the middle of the algorithm

**Review by iulian03 — COMMENTED** [review]
Looking good. I've added a single comment.

File: MangoPay/Libraries/ApiOAuth.php (hunk 1 above)
Comment (williamdes) [inline MangoPay/Libraries/ApiOAuth.php:33]:
```suggestion
```

**Comment by williamdes** [comment]
> Looking good. I've added a single comment.

Thanks, I removed it.
That said, it was important for phpstan analysis since it can not figure out what type is the object.

**Comment by williamdes** [comment]
Hi @iulian03 
Does this PR look okay now ?

**Review by iulian03 — APPROVED** [review]

**Pull request merged**